python/paddle/fluid/layers/nn.py

```diff
@@ -5036,6 +5037,45 @@ def gather(input, index):
     return out
 
 
+def scatter(input, index, updates):
+    """
+    **Scatter Layer**
+
+    Output is obtained by updating the input on selected indices on the first
+    axis.
+
+    .. math::
+
+        Out = X
+        Out[Ids] = Updates
+
+    Args:
+        input: The source input with rank>=1.
+        index: The index input with rank=1.
```

Comment:
Please tell user the type for index, int32_t or int64_t

Reply:
Done